python/src/vmaf/core/executor.py

```diff
@@ -377,7 +378,7 @@ def _post_process_result(cls, result):
     def _get_log_file_path(self, asset):
         return "{workdir}/{executor_id}_{str}".format(
             workdir=asset.workdir, executor_id=self.executor_id,
-            str=hashlib.sha1(str(asset)).hexdigest())
+            str=hashlib.sha1(str(asset).encode("utf-8")).hexdigest())
```

Comment:
`hashlib` requires a `bytes` object in python3
In python2, there was an amalgamation/sometimes-confusion of str/unicode/bytes.